examples/src/main/java/UpdateNftsMetadataExample.java

```diff
@@ -76,19 +76,23 @@ private UpdateNftsMetadataExample() throws Exception {
     }
 
     private void updateNftsMetadata(TokenCreateTransaction tokenCreateTransaction) throws Exception {
+        //Create a non-fungible token (NFT) with the metadata key field set
```

Comment:
I recommend adding a space between `//` and first character  to improve readability for this and the following comments 
```suggestion
        // Create a non-fungible token (NFT) with the metadata key field set
```